.travis.yml

```diff
@@ -1,28 +1,32 @@
 language: python
-
 python:
-  - "3.5"
-
+- 3.5
+- 3.6
 env:
-  - DJANGO=1.10
-  - DJANGO=1.11
-
+- DJANGO=1.11
+- DJANGO=2.0
+- DJANGO=2.1
+- DJANGO=2.2
+- DJANGO=3.0
```

Comment:
this is cool; does travis automatically create a matrix from this?

Reply:
Yes! We're using it on a few of our other projects too

Reply:
awesome!